packages/gmap-vue/examples/heatmap-layer.html

```diff
@@ -0,0 +1,39 @@
+<body>
+  <div id="root">
+    <gmap-map ref="mapRef" :zoom="7" :center="center" map-type-id="roadmap" style="width: 100%; height: 500px">
+        <gmap-marker v-for="(m, index) in markers" :key="index" :position="m.location" :clickable="true" :draggable="true" @click="center=m.location" />
+        <gmap-heatmap :data="markers" :options="{maxIntensity: 120, dissipating: false}" />
+    </gmap-map>
+  </div>
+
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/vue/2.5.17/vue.js"></script>
+  <script src="gmap-vue.js"></script>
+
+  <script>
+
+    Vue.use(VueGoogleMaps, {
+      load: {
+        key: 'AIzaSyDf43lPdwlF98RCBsJOFNKOkoEjkwxb5Sc',
+        libraries: 'visualization'
+      },
+    });
+
+    document.addEventListener('DOMContentLoaded', function () {
+      new Vue({
+        el: '#root',
+        data: {
+          center: {lat:4.5, lng: 99},
+          markers: [
+            { location: new google.maps.LatLng({ lat: 3, lng: 101 }), weight: 100 },
+            { location: new google.maps.LatLng({ lat: 5, lng: 99 }), weight: 50  },
+            { location: new google.maps.LatLng({ lat: 6, lng: 97 }), weight: 80 },
+          ],	
+        },
+        methods: {
+        }
```

Comment:
```suggestion
        methods: {
          setMarkers() {
            this.markers = [
              { location: new google.maps.LatLng({ lat: 3, lng: 101 }), weight: 100 },
              { location: new google.maps.LatLng({ lat: 5, lng: 99 }), weight: 50  },
              { location: new google.maps.LatLng({ lat: 6, lng: 97 }), weight: 80 },
            ];
          }
        },
```
Add a method to load the markers when the page is ready.